src/Annotator.js

```diff
@@ -96,23 +96,22 @@ class Annotator extends EventEmitter {
             this.container = document.querySelector(this.container);
         }
 
+        if (!this.container) {
```

Comment:
as discussed, break this out to another change.

Reply:
https://github.com/box/box-annotations/pull/337